.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-1.23.0-20211220.1904.cjs
```

Comment:
Apologies for the dumb question, but anyways: This means one needs to have `yarn` installed globally, then that `yarn` executable will look at that this file, read the `yarnPath` key, and just execute (or shell out to) this binary instead, and nothing more?

If this is correct, then devs (and CI I guess) would just install _any_ Yarn version (assuming all versions have the same behavior of reading `.yarnrc.yml` and reading `yarnPath` from it)?

Reply:
yes I believe that is accurate